src/middlewared/middlewared/plugins/nfs.py

```diff
@@ -14,6 +14,7 @@
 import middlewared.sqlalchemy as sa
 from middlewared.utils.asyncio_ import asyncio_map
 from middlewared.plugins.nfs_.utils import get_domain, leftmost_has_wildcards, get_wildcard_domain
+from pathlib import PurePath
```

Comment:
For standard library modules, we like to keep these imports at the top of the file.